pkg/server/plugin/nodeattestor/k8spsat/psat.go

```diff
@@ -225,10 +226,6 @@ func (p *AttestorPlugin) Configure(_ context.Context, req *configv1.ConfigureReq
 		return nil, status.Error(codes.InvalidArgument, "core configuration missing trust domain")
 	}
 
-	if len(hclConfig.Clusters) == 0 {
```

Comment:
Not having any clusters defined would be a misconfiguration for (most?) users that aren't dynamically adjusting the list. To aid debuggability, can you change this to emit a warning log instead? The log warning should be emitted down below, just before the call to p.setConfig, so that we only emit the warning when the configuration is actually going to change.

Reply:
A WARNING log message is appropriate.  Expect an update today.